### DISCUSSION TITLE
Hide list view columns that a user don't have access to

### DISCUSSION THREAD

**Discussion by dsod** [payloadcms/payload#848] [Feature Requests & Ideas]
Fields that a user do not have access to, are still shown in the admin interface.
Example:
```typescript
export const Organizations: CollectionConfig = {
  slug: "organizations",
  access: {
    read: ({ req: { user } }) => isSuperUser(user) || isPartner(user),
  },
  fields: [
    {
      name: "type",
      type: "select",
      label: "Type",
      options: [
        organizationTypes.superUsers,
        organizationTypes.partner,
        organizationTypes.customer,
      ],
      access: {
        read: ({ req: { user } }) => isSuperUser(user),
      },
    },
  ]
```

If I log in to the admin interface as a "partner" user, I still see the "Type" column in the Organizations collection list view. This could probably he hidden, as they would never be able to see a value.

With the risk of making the discussion to broad; I also think that documents that a user does not have access to, should display an empty value instead of a text like "Untitled - ID: 62e06d4bf2f20c8efff9005d", i.e.:
![image](https://user-images.githubusercontent.com/19597385/181626663-c678dc24-0570-4771-b8b9-e99883f5f2d1.png)
Example above is a 'relationship' field, where the user don't have access to the specific relation document on this row.

**Comment by XGZepto** [comment]
Update: It's 17 Aug 2023 and we still don't have a follow up :(
